src/api/spec/controllers/webui/staging/workflows_controller_spec.rb

```diff
@@ -2,7 +2,10 @@
 
 RSpec.describe Webui::Staging::WorkflowsController do
   let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:managers_group) {create(:group)}
+  let(:other_managers_group) {create(:group)}
   let(:project) { user.home_project }
+  let(:staging_workflow) { project.create_staging(managers: managers_group) }
```

Comment:
I think we should use a factor here instead and adapt the Factory as well. We do it like this everywhere.

See https://github.com/openSUSE/open-build-service/pull/6257/commits/d867d2953e4f25038f9ff9d08670074c2d609c8a